assets/js/localization.js

```diff
@@ -381,3 +397,21 @@ function getLangByCode(code) {
         return code;
     }
 }
+
+function pickRecaptchaLanguage(locale2) {
```

Comment:
This isn't really picking a language, it's returning a source. How about `getRecaptchaSrc`.